editions/tw5.com/tiddlers/widgets/GenesisWidget.tid

```diff
@@ -39,3 +40,12 @@ src="""\define my-banner(mode:"inline",caption)
 
 <<my-banner mode:"block" caption:"I'm in a DIV">>
 """/>
+
+! Important
+<<.warning """In the following example the parameter `one` is absent from the HTML output, since HTML parameters with the prefix `on` are ''removed for security reasons''. It's needed to remove eg: `onClick=` from "copy / pasted" HTML content. At the moment, every parameter prefixed `on` is removed. 
```

Comment:
Why is "removed for security reasons" in bold type? These aren't parameters, they are attributes, and it is important to use the right terminology.

The explanation here seems awkward. I think we need to cover this properly in "HTML in WikiText". The genesis widget should just note that attributes starting with `on` are suppressed for security reasons, with a link to "HTML in WikiText" for more information.

```